solitude/utils.py

```diff
@@ -2,6 +2,22 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
+# Note deliberately ignoring o, O, 0, i and 1 as too similar.
```

Comment:
might want to ignore `l` (lowercase L) while you're at it since that looks like a 1